service/src/main/java/bio/terra/workspace/service/workspace/flight/create/workspace/CreateWorkspaceV2Flight.java

```diff
@@ -57,30 +69,40 @@ public CreateWorkspaceV2Flight(FlightMap inputParameters, Object applicationCont
           new CreateWorkspacePoliciesStep(
               workspace, policyInputs, appContext.getTpsApiDispatch(), userRequest),
           serviceRetryRule);
+
+      addStep(
```

Comment:
I moved this up earlier, to match the original `[WorkspaceCreateFlight](https://github.com/DataBiosphere/terra-workspace-manager/blob/47dcbd5918e76349165a16230ec9369cf0de9819/service/src/main/java/bio/terra/workspace/service/workspace/flight/create/workspace/WorkspaceCreateFlight.java#L65)`. It's not clear to me that this should ONLY be done for a MC_WORKSPACE.

Reply:
@marctalbott @aherbst-broad I'm particularly interested on your thoughts here-- if this should stay within the `MC_WORKSPACE` case. A RAWLS stage workspace won't have a spendProfileId, will it?

Reply:
Rawls workspaces don't have spend profiles or cloud contexts. If the step is robust to the spend profile ID and cloud context not being present it's probably OK, but it'd be best to avoid that check altogether to avoid spamming as you propose. 

I am unsure about the OOO of linking spend profile attributes before the source workspace attributes. My instinct is that it doesn't matter -- if they are incompatible the merge step will fail out. We do want to ensure that rawls workspaces are having their policies merged from the source workspace, but the OOO doesn't matter in that case since they don't have a spend profile.

Reply:
Added the "if" check, like we had in the original `WorkspaceCreateFlight`.